src/plugins/console/public/lib/mappings/mappings.js

```diff
@@ -54,20 +51,23 @@ export function expandAliases(indicesOrAliases) {
   if (typeof indicesOrAliases === 'string') {
     indicesOrAliases = [indicesOrAliases];
   }
```

Comment:
Since it's a new condition comparing to previous code, can you elaborate on the use case of returning noting, when `indicesOrAliases` is not a string and not an Array?

Reply:
https://github.com/opensearch-project/OpenSearch-Dashboards/issues/3800